Example/ViewController.swift

```diff
@@ -111,5 +111,57 @@ class ViewController: UIViewController {
             print("Finished with selections: \(assets)")
         })
     }
+    
+    @IBAction func showImagePickerWithLimit(_ sender: Any) {
+        let imagePicker = ImagePickerController()
+        imagePicker.settings.selection.max = 5
+        imagePicker.settings.theme.selectionStyle = .numbered
+        imagePicker.settings.fetch.assets.supportedMediaTypes = [
+            .image(max: 2),
+            .video(max: 1)
+        ]
+
+        let start = Date()
+        imagePicker.imagePickerDelegate = self
+        self.presentImagePicker(imagePicker, select: { (asset) in
```

Comment:
If you don't want to use the closure API. Just present it as you normally would with any view controller.

Reply:
i understand but if you set delegate with 
```
            imagePicker.imagePickerDelegate = imagePicker
```
it should override my own delegate from Example/ViewController, how to use delegate API instead of closure API

Reply:
> Just present it as you normally would with any view controller.

```
self.present(imagePicker, animated: animated)
```